integration/integration_test.go

```diff
@@ -164,6 +169,7 @@ func readSpdxJson(t *testing.T, filePath string) *spdx.Document2_2 {
 	// We don't compare values which change each time an SBOM is generated
 	bom.CreationInfo.Created = ""
 	bom.CreationInfo.DocumentNamespace = ""
+	bom.Relationships = nil
```

Comment:
We should ensure `Relationships` is correct. Can we sort it?

Reply:
I confused myself. Thanks.
I added sorting by `ElementRefID` for this.